Modules/Invoice/Services/InvoiceService.php

```diff
@@ -205,9 +205,9 @@ private function formatInvoicesForExport($invoices)
         return $invoices->map(function ($invoice) {
             return [
                 'Project' => $invoice->project->name,
-                'Amount' => $invoice->display_amount,
+                'Amount' => $invoice->amount,
                 'GST' => $invoice->gst,
-                'Amount (+GST)' => $invoice->invoiceAmount(),
+                'Amount (+GST)' => (float) str_replace(['$', '₹'], '', $invoice->invoiceAmount()),
```

Comment:
```suggestion
                'Amount (+GST)' => $invoice->amount + $invoice->gst,
```

Can you once try this suggestion and compare the result.

Reply:
Sure

Reply:
It works but it is used in multiple places should I changed it everywhere. But the code will be repeated

Reply:
Just use it here only for now.